File: src/ua/inf/krre/aprilbrush/MainActivity.java
package ua.inf.krre.aprilbrush;

import ua.inf.krre.aprilbrush.logic.BrushEngine;
import android.app.Activity;
import android.os.Bundle;
import android.view.Menu;
import android.view.MotionEvent;

public class MainActivity extends Activity {
    private BrushEngine engine;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main);

        engine = new BrushEngine();
    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        // Inflate the menu; this adds items to the action bar if it is present.
        getMenuInflater().inflate(R.menu.main, menu);
        return true;
    }

    @Override
    public boolean onTouchEvent(MotionEvent event) {
        engine.paintDab(event);
        return true;
    }
}

File: AprilBrush/src/main/kotlin/activity/MainActivity.kt
package org.krre.aprilbrush.activity

import android.app.Activity
import android.os.Bundle
import org.krre.aprilbrush.R
import android.util.Log
import android.app.ActivityManager
import android.content.Context

public class MainActivity() : Activity() {
    private val TAG = "AB"
    public var memoryClass : Int? = 0
        get
        private set

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val activityManager = getBaseContext()?.getSystemService(Context.ACTIVITY_SERVICE) as ActivityManager?
        memoryClass = activityManager?.getLargeMemoryClass()
    }
}
